modules/API/pages/gsql-endpoints.adoc

```diff
@@ -545,7 +548,7 @@ Sample Request::
 --
 [source,bash]
 ----
-curl -X DELETE -H "content-type: text/plain" -u tigergraph:tigergraph "http://localhost:14240/gsql/v1/schema/indexes/nameIndex"
+curl -X DELETE -H "content-type: text/plain" -u tigergraph:tigergraph "http://localhost:14240/gsql/v1/schema/indexes/nameIndex?vertex=Account"
```

Comment:
Fixing a doc error for an existing endpoint.
vertex is a required parameter, so the example was incorrect.